packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/inspection/responses.ts

```diff
@@ -0,0 +1,188 @@
+import { RDD } from '@chainlink/gauntlet-terra'
+import { InspectInstruction, instructionToInspectCommand } from '../../../abstract/inspectionWrapper'
+import { CONTRACT_LIST } from '../../../../lib/contracts'
+import { CATEGORIES } from '../../../../lib/constants'
+import { LCDClient } from '@terra-money/terra.js'
+import { getLatestOCRNewTransmissionEvent, RoundData } from '../../../../lib/inspection'
+import { inspection, logger } from '@chainlink/gauntlet-core/dist/utils'
+import { dateFromUnix } from '../../../../lib/utils'
+
+type CommandInput = {
+  transmitters: string[]
+  description: string
+  aggregatorOracles: Oracle[]
+}
+
+type ContractExpectedInfo = {
+  transmitters: string[]
+  description: string
+  aggregatorAddress: string
+  latestRoundData: RoundData
+  event: NewTransmissionEventData
+}
+
+type Oracle = {
+  transmitter: string
+  name: string
+  apis: string[]
+  observation?: number
+}
+
+type NewTransmissionEventData = {
+  answer: number
+  transmitter: string
+  observations: number[]
+  observers: string
+  observations_timestamp: number
+  aggregator_round_id: number
+}
+
+const makeInput = async (flags: any, args: string[]): Promise<CommandInput> => {
+  if (flags.input) return flags.input as CommandInput
+  const rdd = RDD.getRDD(flags.rdd)
+  const operators = rdd.operators
+  const aggregatorProps = rdd.contracts[args[0]]
+  const oracles = aggregatorProps['oracles']
+  const aggregatorOperators: string[] = oracles.map((o) => o.operator)
+  const transmitters = aggregatorOperators.map((o) => rdd.operators[o].ocrNodeAddress[0])
+
+  const aggregatorOracles = oracles.map((o) => {
+    return {
+      transmitter: operators[o.operator].ocrNodeAddress[0],
+      name: o.operator,
+      apis: o.api,
+    }
+  })
+
+  return {
+    transmitters,
+    description: aggregatorProps.name,
+    aggregatorOracles,
+  }
+}
+
+const makeOnchainData = (provider: LCDClient) => async (
+  instructionsData: any[],
+  input: CommandInput,
+  aggregator: string,
+): Promise<ContractExpectedInfo> => {
+  const latestRoundData = instructionsData[0]
+  const transmitters = instructionsData[1]
+  const description = instructionsData[2]
+
+  let event
```

Comment:
Be careful here, event will have an `any` type here, meaning that you can include anything you want there. A way to fix this could be:
```js
      ...
      transmissionTimestamp: latestRoundData.transmission_timestamp,
    },
    ...(onchainEvent && {
      event: {
        answer: Number(onchainEvent.answer[0]),
        transmitter: onchainEvent.transmitter[0],
        observations: onchainEvent.observations.map((o) => Number(o)),
        observers: onchainEvent.observers[0],
        observations_timestamp: Number(onchainEvent.observations_timestamp[0]),
        aggregator_round_id: Number(onchainEvent.aggregator_round_id[0]),
      },
    }),
```
And marking `event` as optional in `ContractExpectedInfo`